tests/components/hikvision/test_init.py

```diff
@@ -89,3 +89,19 @@ async def test_setup_entry_no_device_id(
     await hass.async_block_till_done()
 
     assert mock_config_entry.state is ConfigEntryState.SETUP_RETRY
+
+
+async def test_setup_entry_nvr_fetches_events(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    mock_hikcamera: MagicMock,
+) -> None:
+    """Test setup fetches NVR events for NVR devices."""
+    mock_hikcamera.return_value.get_type = "NVR"
+    mock_hikcamera.return_value.get_event_triggers.return_value = {"Motion": [1, 2]}
```

Comment:
How often will you test with the NVR? You could also extend the `mock_hikcamera` and create a `mock_hik_nvr` that imports `mock_hikcamera` and sets the right fields for the NVR. This way you don't have to repeat yourself